build.gradle

```diff
@@ -473,6 +479,15 @@ task mapNamedJar(dependsOn: [buildTinyWithEnum, mapIntermediaryJar]) {
 	}
 }
 
+task decompileCFR(dependsOn: mapNamedJar, type: JavaExec) {
+	doFirst {
+		file('src').deleteDir()
+	}
+
+	classpath = configurations.cfrRuntime
+	main = 'org.benf.cfr.reader.Main'
+	args '--outputdir', 'src', '--extraclasspath', Stream.of(libraries.listFiles()).map{it.name}.collect(Collectors.joining(";")), namedJar.getAbsolutePath()
```

Comment:
```groovy
Stream.of(libraries.listFiles()).map{it.name}.collect(Collectors.joining(";"))
```
could just be
```groovy
libraries.listFiles().collect { it.name }.join ';'
```

Reply:
yeah, thats better but it doesnt solve the first issue of the libs not being there. ive removed adding the libs as the code looks fine without it (I havent been able to diff it just yet)